test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -1219,7 +1219,7 @@ protected TableMetadata currentTableMetadata()
         return Schema.instance.getTableMetadata(KEYSPACE, currentTable());
     }
 
-    protected com.datastax.driver.core.ResultSet executeNet(ProtocolVersion protocolVersion, String query, Object... values) throws Throwable
+    protected com.datastax.driver.core.ResultSet executeNet(ProtocolVersion protocolVersion, String query, Object... values)
```

Comment:
I like the cleanup of unnecessary / redundant "throws X" lines in tests, but to check: what was the thinking of doing this as part of this patch rather than having a separate patch for it since it seems unrelated to the primary ALLOW FILTERING guardrails work?

Reply:
I think this was from @maedhroz ; I welcome it in this patch or external one... I just wants it =D

Reply:
https://youtu.be/Uo_TRFXh5XI

Reply:
I think I'd prefer a separate patch so we can do a proper cleanup of the thousands of unused `throws Throwable` parts that it leaves around the tests, many of them transitive.